src/SecurityCompare/SecurityCompareConnector.ts

```diff
@@ -0,0 +1,176 @@
+import { SecurityDetailsConverter } from '../SecurityDetails';
+import {
+    TrailingPerformanceConverter,
+    AssetAllocationsConverter,
+    CountryExposureConverter,
+    RegionalExposureConverter,
+    GlobalStockSectorBreakdownConverter
+} from '../SecurityDetails/Converters';
+import {
+    SecurityCompareOptions,
+    SecurityCompareMetadata
+} from '../SecurityDetails/SecurityDetailsOptions';
+import MorningstarAPI from '../Shared/MorningstarAPI';
+import MorningstarConnector from '../Shared/MorningstarConnector';
+import MorningstarURL from '../Shared/MorningstarURL';
+
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+
+export class SecurityCompareConnector extends MorningstarConnector {
+
+
+    /* *
+     *
+     *  Constructor
+     *
+     * */
+
+
+    public constructor (
+        options: SecurityCompareOptions
+    ) {
+        super(options);
+
+        switch (options.converter?.type) {
+            case 'TrailingPerformance':
+            default:
+                this.converter = new TrailingPerformanceConverter({
+                    ...options.converter
+                });
+                break;
+            case 'AssetAllocations':
+                this.converter = new AssetAllocationsConverter({
+                    ...options.converter
+                });
+                break;
+            case 'CountryExposure':
+                this.converter = new CountryExposureConverter({
+                    ...options.converter
+                });
+                break;
+            case 'RegionalExposure':
+                this.converter = new RegionalExposureConverter({
+                    ...options.converter
+                });
+                break;
+            case 'GlobalStockSectorBreakdown':
+                this.converter = new GlobalStockSectorBreakdownConverter({
+                    ...options.converter
+                });
+                break;
+        }
+
+        this.metadata = this.converter.metadata;
+        this.options = options;
+    }
+
+
+    /* *
+     *
+     *  Properties
+     *
+     * */
+
+
+    public override readonly converter: SecurityDetailsConverter;
+
+
+    public override readonly metadata: SecurityCompareMetadata;
+
+
+    public override readonly options: SecurityCompareOptions;
+
+
+    /* *
+     *
+     *  Functions
+     *
+     * */
+
+
+    public override async load (
+        options?: SecurityCompareOptions
+    ): Promise<this> {
+
+        await super.load();
+
+        const userOptions = { ...this.options, ...options };
+        const { security, viewIds = 'MFsnapshot' } = userOptions;
+        const { ids: securityIds, idType: securityIdType } = (security || {});
+        const api = this.api = this.api || new MorningstarAPI(userOptions.api);
+        const url = new MorningstarURL('ecint/v1/multi-securities/', api.baseURL);
+
+        const searchParams = url.searchParams;
+        searchParams.set('ids', '' + securityIds.join('|'));
```

Comment:
@bm64 should we extract UTF_PIPE variable to a common place? Currently stored in Screener, perhaps better make it a utils const to reuse.

Reply:
Yes, I think we should do that, and same goes for `UTF_COLON`.

Reply:
So last request from us @DJTechnoo :D